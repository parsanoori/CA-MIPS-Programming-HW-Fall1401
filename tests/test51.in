42
760182028
652618156
430025639
497625809
917289207
848726897
143153211
480909647
355071629
381957871
407865983
968081606
20705174
854098756
10540040
775406178
246692608
60292112
609392586
619294529
569814980
899075281
830112213
333675956
978984939
1073486065
412099110
355479657
830039808
469745165
432317238
239282587
176270769
86094160
731567977
212306711
268151104
777594867
578838962
404973092
1045081587
850178239