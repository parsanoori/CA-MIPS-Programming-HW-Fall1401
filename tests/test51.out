10540040
20705174
60292112
86094160
143153211
176270769
212306711
239282587
246692608
268151104
333675956
355071629
355479657
381957871
404973092
407865983
412099110
430025639
432317238
469745165
480909647
497625809
569814980
578838962
609392586
619294529
652618156
731567977
760182028
775406178
777594867
830039808
830112213
848726897
850178239
854098756
899075281
917289207
968081606
978984939
1045081587
1073486065